Podcast/SeriesDetailHeaderView.swift

```diff
@@ -61,11 +62,12 @@ class SeriesDetailHeaderView: UIView {
     var settingsButton: UIButton!
     var shareButton: UIButton!
     
+    var series: Series?
```

Comment:
a view should not hold onto a model for good MVC design.

Reply:
There should be a better way to do this, worst case we can hold onto just the Tags but with a datasource, delegate protocol you should be able to create a good design for this that does not hold onto any models